language-server/test/dotty/tools/languageserver/util/actions/CodeDefinition.scala

```diff
@@ -0,0 +1,16 @@
+package dotty.tools.languageserver.util.actions
+
+import dotty.tools.languageserver.util._
+import dotty.tools.languageserver.util.embedded.CodeMarker
+
+class CodeDefinition(val range: CodeRange, refOpt: Option[CodeRange]) extends ActionOnRange {
+
+  override def onMarker(marker: CodeMarker): Exec[Unit] = {
+    val res = server.definition(fix(marker.toTextDocumentPositionParams)).get()
+    assert(res.size() == refOpt.size, res)
```

Comment:
`size()` -> `size`

Reply:
`res` is a java collection, hence it requires `size()`.

Reply:
It was complaining or waring at the time. I will double check.